prereise/gather/demanddata/bldg_electrification/floor_area_avg_slopes.py

```diff
@@ -480,13 +594,58 @@ def main_plots(iso, zone_shape, pumas_shp, state_shp, plot_show=True):
     zone_shp_vt = state_shp_overlay("VT", state_shp, zone_shape)
     zone_shp_ct = state_shp_overlay("CT", state_shp, zone_shape)
     zone_shp_ri = state_shp_overlay("RI", state_shp, zone_shape)
+    zone_shp_de = state_shp_overlay("DE", state_shp, zone_shape)
+    zone_shp_il = state_shp_overlay("IL", state_shp, zone_shape)
+    zone_shp_in = state_shp_overlay("IN", state_shp, zone_shape)
+    zone_shp_ky = state_shp_overlay("KY", state_shp, zone_shape)
+    zone_shp_md = state_shp_overlay("MD", state_shp, zone_shape)
+    zone_shp_tn = state_shp_overlay("TN", state_shp, zone_shape)
+    zone_shp_nc = state_shp_overlay("NC", state_shp, zone_shape)
+    zone_shp_mi = state_shp_overlay("MI", state_shp, zone_shape)
+    zone_shp_nj = state_shp_overlay("NJ", state_shp, zone_shape)
+    zone_shp_oh = state_shp_overlay("OH", state_shp, zone_shape)
+    zone_shp_pa = state_shp_overlay("PA", state_shp, zone_shape)
+    zone_shp_va = state_shp_overlay("VA", state_shp, zone_shape)
+    zone_shp_wva = state_shp_overlay("WV", state_shp, zone_shape)
+    zone_shp_dc = state_shp_overlay("DC", state_shp, zone_shape)
+    zone_shp_ks = state_shp_overlay("KS", state_shp, zone_shape)
+    zone_shp_ok = state_shp_overlay("OK", state_shp, zone_shape)
+    zone_shp_nm = state_shp_overlay("NM", state_shp, zone_shape)
+    zone_shp_tx = state_shp_overlay("TX", state_shp, zone_shape)
+    zone_shp_ar = state_shp_overlay("AR", state_shp, zone_shape)
+    zone_shp_la = state_shp_overlay("LA", state_shp, zone_shape)
+    zone_shp_mo = state_shp_overlay("MO", state_shp, zone_shape)
+    zone_shp_sd = state_shp_overlay("SD", state_shp, zone_shape)
+    zone_shp_nd = state_shp_overlay("ND", state_shp, zone_shape)
+    zone_shp_mt = state_shp_overlay("MT", state_shp, zone_shape)
+    zone_shp_mn = state_shp_overlay("MN", state_shp, zone_shape)
+    zone_shp_ia = state_shp_overlay("IA", state_shp, zone_shape)
+    zone_shp_wy = state_shp_overlay("WY", state_shp, zone_shape)
+    zone_shp_ne = state_shp_overlay("NE", state_shp, zone_shape)
+    zone_shp_ms = state_shp_overlay("MS", state_shp, zone_shape)
+    zone_shp_wi = state_shp_overlay("WI", state_shp, zone_shape)
+    zone_shp_az = state_shp_overlay("AZ", state_shp, zone_shape)
+    zone_shp_co = state_shp_overlay("CO", state_shp, zone_shape)
+    zone_shp_nv = state_shp_overlay("NV", state_shp, zone_shape)
+    zone_shp_ca = state_shp_overlay("CA", state_shp, zone_shape)
+    zone_shp_wa = state_shp_overlay("WA", state_shp, zone_shape)
+    zone_shp_or = state_shp_overlay("OR", state_shp, zone_shape)
+    zone_shp_id = state_shp_overlay("ID", state_shp, zone_shape)
+    zone_shp_ut = state_shp_overlay("UT", state_shp, zone_shape)
+    zone_shp_al = state_shp_overlay("AL", state_shp, zone_shape)
+    zone_shp_ga = state_shp_overlay("GA", state_shp, zone_shape)
+    zone_shp_sc = state_shp_overlay("SC", state_shp, zone_shape)
+    zone_shp_fl = state_shp_overlay("FL", state_shp, zone_shape)
```

Comment:
Similarly, another option here is creating a dictionary rather than listing everything out: `zone_shp_state = {s : state_shp_overlay(s, state_shp, zone_shape) for s in state_list}`

Reply:
Made the changes